src/react/src/components/Dashboard/ConfirmActionDialog.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState, useMemo } from 'react';
```

Comment:
Maybe it is better to rename this component to `RejectActionDialog.jsx` or `RejectModerationEventDialog.jsx`?

Reply:
Initially, I thought that this component would be common for all moderation event actions, therefore I called it `ConfirmActionDialog`.
But in the current stage, I agree that it would be better to rename it because it's unknown whether we will implement such a dialog window for other actions or not.

Renamed to the `RejectModerationEventDialog`.
Thank you.